play/src/main/scala/org/lyranthe/prometheus/client/play/filters/PrometheusFilter.scala

```diff
@@ -0,0 +1,57 @@
+package org.lyranthe.prometheus.client.play.filters
+
+import org.lyranthe.prometheus.client.{play => _, _}
+import org.lyranthe.prometheus.client.histogram.LabelledHistogram
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+class PrometheusFilter()(implicit val registry: Registry, executionContext: ExecutionContext) extends Filter {
+
+  private val ClientErrorClass = "4xx"
+
+  private val ServerErrorClass = "5xx"
+
+  private val RouteRegex = "[/a-zA-Z0-9$_\\-]+".r
```

Comment:
What do you think of anchoring the regex - start with `^`, end with `$`? Means we must match the full path, so we don't accidentally miss parts of the path due to a regex error.